src/opower/opower.py

```diff
@@ -149,7 +150,7 @@ async def async_login(self) -> None:
         """
         try:
             self.access_token = await self.utility.async_login(
-                self.session, self.username, self.password
+                self.session, self.username, self.password, self.optional_auth

```

Comment:
> The wiring isn't quite complete here. Adding the optional_auth currently breaks all the other utilities since they're not expecting that parameter. Do we just add the param to those and don't use it? I'm not quite sure what the proper way to do this in Python is.

Python has `**kwargs: Any` but that means each utility would need to return what additional fields it accepts so that UI can ask for them. Given that 2FA will likely become the norm for all utilities, I'm OK passing it in all utilities and ignoring it.